modules/virt-adding-configmap-for-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-adding-configmap-for-obsolete-cpu-models.adoc
+
+[id="virt-adding-configmap-for-obsolete-cpu-models_{context}"]
+= Adding a ConfigMap for obsolete CPU models
```

Comment:
Our OCP doc, where ConfigMaps are explained, uses the gerund "Adding..." in the title and I was following that usage pattern in the title headings:
https://docs.openshift.com/container-platform/4.5/virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.html#virt-adding-secret-configmap-service-account-to-vm_virt-managing-configmaps-secrets-service-accounts

If there is a reference that recommends avoiding the use of gerund "Adding" and the confusion it causes to non-native English speakers, I would like to add this to my awareness and avoid using the gerund "Adding" to title headings in the future! Thanks!

Reply:
To clarify, I think that if you use "Adding," there should be a "to..." prepositional phrase. Otherwise, it's ambiguous what you're adding the ConfigMap to. Ambiguities like that can be especially confusing for non-native speakers, especially when a word has more than one meaning.

In the title you cited, the user is adding a ConfigMap (that already exists) "to a virtual machine." Since you're creating a new ConfigMap, I think it would make more sense to start with "Creating," especially because I don't see a step where you actually add the ConfigMap to something else.

Reply:
New module is:
virt-configuring-configmap-for-obsolete-cpu-models